kumascript/macros/PreviousMenuNext.ejs

```diff
@@ -37,29 +35,34 @@ var s_Menu = mdn.localString({
     "zh-TW": " 概述：",
 });
 
-if ($0) {
-    strPrevious = '<li><a class="button secondary" href="/' + lang + '/docs/' + $0.replace(/ /g, '_') + '"><span class="button-wrap">' + s_PreviousNext[0] + '</span></a></li>';
+function generateLink(pageSlug, linkText) {
+  return `<li><a class="button secondary" href="/${locale}/docs/${pageSlug}"><span class="button-wrap">${linkText}</span></a></li>`;
 }
 
-if ($1) {
-    strNext = '<li><a class="button secondary" href="/' + lang + '/docs/' + $1.replace(/ /g, '_') + '"><span class="button-wrap">' + s_PreviousNext[1] + '</span></a></li>';
+function generateMenuLink(pageSlug) {
+  const url = `/${locale}/docs/${pageSlug}`;
+  let aPage = info.getPageByURL(url);
+  if (!aPage.url && locale !== defaultLocale) {
```

Comment:
Have you got any examples where the menu link isn't localized? Reading the source for `info.getPageByURL` it seems that'll return an empty object, but it'll be best to test

Reply:
Doesn't this happen when a newly added page is not yet translated?

Reply:
I wanted to test that it worked, did so locally by just deleting a localized menu page (e.g. fr/learn/accessibility/index.md) and then visiting a subpage (e.g. http://localhost:3000/fr/docs/Learn/Accessibility/HTML) - it works fine!